guides/common/modules/con_introduction-to-application-life-cycles.adoc

```diff
@@ -31,10 +31,14 @@ When the test server passes the initial phase, you can set the production-level
 Another example is a development life cycle for a software product.
 To develop a new piece of software in a development environment, test it in a quality assurance environment, pre-release as a beta, then release the software as a production-level application.
 
-ifndef::satellite[]
+ifdef::foreman-el,foreman-deb,katello[]
 image::application_life_cycle.png[title="The {Project} Application Life Cycle", alt="The {Project} Application Life Cycle"]
 endif::[]
 
 ifdef::satellite[]
 image::application_life_cycle_sat.png[title="The {Project} Application Life Cycle", alt="The {Project} Application Life Cycle"]
 endif::[]
+
+ifdef::orcharhino[]
+image::common/application_life_cycle_orcharhino.svg[title="The {Project} Application Life Cycle", alt="The {Project} Application Life Cycle"]
```

Comment:
How does this differ from the existing PNG, other than the file format? Can't we use this image for both katello and orcharhino?

Reply:
Both files use slightly different shades of blue: one for orcharhino (acccording to our internal style guide), one I got from theforeman.org.

Reply:
To me the shades of blue are not that important and less duplication is less maintenance work and thus better. I'll merge it now, but perhaps @Lennonka can weigh in on this.